Readability.js

```diff
@@ -434,6 +434,24 @@ Readability.prototype = {
     return node && node.nextElementSibling;
   },
 
+  _checkByline: function(node, matchString) {
+    if (this._articleByline) {
+      return false;
+    }
+
+    var rel = null;
+    if (node.getAttribute !== undefined) {
+      rel = node.getAttribute("rel");
```

Comment:
Because this is `var` and not `let` you can just use `var rel =` in the loop and it'll be `undefined` otherwise (which is OK).